xpdacq/xpdacq.py

```diff
@@ -282,6 +282,56 @@ def _insert_collection(collection_name, collection_obj, new_uid=None):
                                collection_name) + '.yaml', 'w') as f:
             yaml.dump(glbl.collection, f)
 
+def set_beamdump_suspender(xrun, suspend_thres=None, resume_thres=None,
+                           wait_time=None, clear=True):
+    """helper function to set suspender based on ring_current
+
+    Parameters
+    ----------
+    xrun : instance of RunEngine
+        the run engine instance suspender will be installed
+    suspend_thres : float, optional
+        suspend if ring current falls below this threshold. default
+        is the larger value between 50 mA or 50% of current ring current
+    resume_thres : float, optional
+        resume if tha ring current ramps higher than this value. default
+        is the larger value among 150 mA or 80% of current ring current
```

Comment:
"suspend if ring current value falls below this threshold. ring current value is read out from ring current signal when set_beamdump_suspender function is executed"

I add these sentences to the docstring. Happy to refine it in the other PR if it is not sufficient